packages/flutter_test/lib/src/_goldens_io.dart

```diff
@@ -240,3 +243,18 @@ ComparisonResult compareLists(List<int> test, List<int> master) {
   }
   return ComparisonResult(passed: true);
 }
+
+/// An unsupported [WebGoldenComparator] that exists for API compatibility.
+class DefaultWebGoldenComparator extends WebGoldenComparator {
+  DefaultWebGoldenComparator(Uri _);
+
+  @override
+  Future<bool> compare(Element element, Size size, Uri golden) {
+    throw UnsupportedError('DefaultWebGoldenComparator is noonly supported on the web.');
```

Comment:
```suggestion
    throw UnsupportedError('DefaultWebGoldenComparator is not supported on the web.');
```

Reply:
Updated. It's actually `only supported` instead of `not supported though.